packages/jobs/src/core/loaders.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'node:path'
+import { pathToFileURL } from 'node:url'
+
+import fg from 'fast-glob'
+
+import { registerApiSideBabelHook } from '@redwoodjs/babel-config'
+import { getPaths } from '@redwoodjs/project-config'
+
+import { JobsLibNotFoundError, JobNotFoundError } from './errors'
+
+const DEV_ENVIRONMENTS = ['development', 'test']
+
+const isDev = DEV_ENVIRONMENTS.includes(process.env.NODE_ENV || '')
```

Comment:
Maybe we can use Vite's `MODE` instead of `NODE_ENV` as a more dependable way of checking for dev vs prod?
https://vitejs.dev/guide/env-and-mode#node-env-and-modes

Reply:
Hmm, isn't that just a build-time thing though? What about once are the files are built and done and you're now serving the app? Since the code that determines where to serve the files from lives in the framework itself, that's all running at runtime... the `MODE` wouldn't be available at that time, would it?